src/components/asthma/components/AsthmaAirQualities/AsthmaAirQualities.css

```diff
@@ -0,0 +1,53 @@
+.mdhui-asthma-air-qualities {
+    padding: 16px;
+}
+
+.mdhui-asthma-air-qualities-header {
+    display: grid;
+    grid-template-columns: auto max-content;
+}
+
+.mdhui-asthma-air-qualities-title {
```

Comment:
Same here this feels like we can just use a <Title> component

Reply:
Yep, done.